src/category_theory/opposites.lean

```diff
@@ -173,4 +178,27 @@ end
 
 end functor
 
+omit 𝒞
```

Comment:
Everything past here is in the wrong place (`omit 𝒞` should indicate that it is no longer category theory!). The right thing is to use #538.

Reply:
Good point! Is there a reason for using separate notions of opposites in algebra and category theory?

Reply:
As far as I know, just that they were being introduced simultaneously by different people.